modules/nixos/where-is-my-sddm-theme.nix

```
@@ -0,0 +1,93 @@
+{ catppuccinLib }:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (builtins) elem;
+  inherit (catppuccinLib) fromINI;
```

You probably want this based on your usage
```suggestion
  inherit (catppuccinLib) importINI;
```